aten/src/THC/THCCachingAllocator.cpp

```diff
@@ -587,3 +587,50 @@ THC_API uint64_t THCCachingAllocator_maxMemoryCached(int device) {
   assertValidDevice(device);
   return caching_allocator.get_stats_for_device(device).max_amount_cached;
 }
+
+//
+// In CUDA IPC, sender sends a tensor to receiver, THCCaching_CUDAIpcDevptr
+// is called by receiver process to get access to the memory where the tensor
```

Comment:
nit: I'd probably say something more like, is called by the receiving process to map the CUDA memory from the sending process into its own address space.